esp/services/ws_access/ws_accessService.cpp

```diff
@@ -2087,24 +2088,27 @@ bool Cws_accessEx::onResourceDelete(IEspContext &context, IEspResourceDeleteRequ
                 if(prefix && *prefix)
                     namebuf.insert(0, prefix);
 
-                secmgr->deleteResource(rtype, namebuf.str(), basednReq->getBasedn(), context.querySecureContext());
+                try
+                {
+                    secmgr->deleteResource(rtype, namebuf.str(), basednReq->getBasedn(), context.querySecureContext());
+                }
+                catch(IException* e)
+                {
+                    me->append(*e);
+                }
             }
         }
-        catch(IException* e)
-        {
-            StringBuffer emsg;
-            e->errorMessage(emsg);
-            resp.setRetcode(e->errorCode());
-            resp.setRetmsg(emsg.str());
-            return false;
-        }
+
         catch(...)
         {
             resp.setRetcode(-1);
             resp.setRetmsg("Unknown error");
             return false;
         }
 
+        if(me->ordinality())
+            throw me.getLink();
```

Comment:
trivial:  throw me.getClear() is slightly more efficient.  Does not matter in this context.